src/object/cli_obj.c

```diff
@@ -2193,7 +2195,6 @@ obj_comp_cb(tse_task_t *task, void *data)
 	obj_auxi = tse_task_stack_pop(task, sizeof(*obj_auxi));
 	obj_auxi->to_leader = 0;
 	obj_auxi->io_retry = 0;
-	obj_auxi->csum_retry = 0;
```

Comment:
if you remove it it means it cannot be clear once it is set, probably with problem for example:
1) fetch first time met a csum err and retry with another shard_B
2) the RPC to that shard get temporary network err for some reason, or just pool map ESTALE, then retry
3) in the retried fetch see that csum retry flag and retry shard_C, if no other shard it will return error to user. but possibly just ESTALE and can be finished refresh pool map and retry again.

Any reason for the removing?